lumeer-core/src/main/java/io/lumeer/core/facade/SuggestionFacade.java

```diff
@@ -98,18 +106,50 @@ private JsonSuggestions suggestAll(String text, int limit) {
       return collections;
    }
 
+   private List<LinkType> suggestLinkTypes(String text, int limit) {
+      List<Collection> collections = collectionDao.getCollections(createSimpleSearchQuery());
```

Comment:
Why this call? Are we making link suggestions based on collection names?

Reply:
We need this for check which link-types can user read.

Reply:
Oh, I see. I forgot about it and did not see checking of user permissions anywhere.

Reply:
Maybe it would be worth naming it `allowedCollections` or something like that.